### PR TITLE
Update Electron help menu to reflect new in-app menu

### PR DESCRIPTION
**Pull request by jfdoming**
Modifies the Electron menu items to match the web app

### CODE REVIEW

File: packages/desktop-electron/menu.ts [hunk 1]
```diff
@@ -173,7 +174,16 @@ export function getMenu(
       role: 'help',
       submenu: [
         {
-          label: 'Keyboard Shortcuts Reference',
+          label: 'Documentation',
+          async click(_menuItem, focusedWin) {
+            const page = await focusedWin?.webContents.executeJavaScript(
+              'window.location.pathname',
```
Comment (jfdoming) [inline packages/desktop-electron/menu.ts]: Open to feedback if there's a more standard way to do this, not too familiar with Electron

File: packages/loot-core/src/client/actions/app.ts [hunk 2]
```diff
@@ -52,3 +52,29 @@ export function reloadApp() {
...
+      return 'https://actualbudget.org/docs/transactions/payees';
+    case '/rules':
+      return 'https://actualbudget.org/docs/budgeting/rules';
+    case '/settings':
+      return 'https://actualbudget.org/docs/settings';
+    default:
+      // All pages under /accounts, plus any missing pages
+      return 'https://actualbudget.org/docs';
+  }
+};
```
[the remark this answers is not in the document]
Comment (jfdoming) [reply packages/loot-core/src/client/actions/app.ts:74]: (discussed in the previous PR for this function)

File: packages/desktop-electron/menu.ts [hunk 3]
```diff
@@ -173,7 +167,15 @@ export function getMenu(
...
-          label: 'Keyboard Shortcuts Reference',
+          label: 'Documentation',
+          click(_menuItem, focusedWin) {
+            focusedWin?.webContents.executeJavaScript(
+              'window.__actionsForMenu && window.__actionsForMenu.openDocsForCurrentPage()',
+            );
+          },
+        },
+        {
+          label: 'Keyboard Shortcuts',
```
[the remark this answers is not in the document]
Comment (jfdoming) [reply packages/desktop-electron/menu.ts:178]: out of scope

**Review by MikesGlitch — APPROVED** [review]
LGTM

**Pull request merged**